exemples/tls-auth/docker-compose.tls-auth.yml

```diff
@@ -0,0 +1,102 @@
+version: "3.8"
+
+services:
+  traefik:
+    image: "traefik:v2.9.4"
+    container_name: "traefik"
+    restart: unless-stopped
+    command:
+      - "--log.level=DEBUG"
+      - "--accesslog"
+      - "--accesslog.filepath=/var/log/traefik/access.log"
+      - "--api.insecure=true"
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+
+      - "--experimental.localplugins.bouncer.modulename=github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - logs-tls-auth:/var/log/traefik
+      - crowdsec-certs-tls-auth:/etc/traefik/crowdsec-certs
+      - ./../../:/plugins-local/src/github.com/maxlerebourg/crowdsec-bouncer-traefik-plugin
+
+    ports:
+      - 80:80
+      - 8080:8080
+    depends_on:
+      - crowdsec
+
+  # whoami-foo:
```

Comment:
remove if not used

Reply:
It is used, this part is not finished yet

I still have a bug with crowdsec client auth.
It's about how the container behave when it start and I cannot update all configuration